packages/core/src/new-api/internal/module-builder.ts

```diff
@@ -220,6 +226,74 @@ export class IgnitionModuleBuilderImplementation<
     return future;
   }
 
+  public call<ContractNameT extends string, FunctionNameT extends string>(
+    contractFuture: ContractFuture<ContractNameT>,
+    functionName: FunctionNameT,
+    args: SolidityParamsType = [],
+    options: CallOptions = {}
+  ): NamedContractCallFuture<ContractNameT, FunctionNameT> {
+    const id = options.id ?? functionName;
+    const futureId = `${this._module.id}:${contractFuture.contractName}:${id}`;
```

Comment:
Nice!

What if we use another symbol for the second `:`? So that users can thing of `:` as the separator between module and future.

For example,

```suggestion
    const futureId = `${this._module.id}:${contractFuture.contractName}#${id}`;
```